src/bundle/Resources/public/js/scripts/login.js

```diff
@@ -0,0 +1,19 @@
+(function (global, doc, eZ) {
+    const passwordInputNode = doc.querySelector('.ez-login__input--password');
+    const viewIconNode = doc.querySelector('.ez-login__password-visibility-toggler .ez-icon--view');
+    const viewHideIconNode = doc.querySelector('.ez-login__password-visibility-toggler .ez-icon--view-hide');
+
+    doc.querySelector('.ez-login__password-visibility-toggler').addEventListener('click', () => {
+        if (passwordInputNode) {
+            let inputTypeToSet = 'password';
+
+            if (passwordInputNode.type.toLowerCase() === 'password') {
```

Comment:
Also I'm not sure you have to do `toLowerCase()`, of course it's safer, but it's our code after all in login page, so maybe just make sure it's already lowercase?